src/Engine/Spells/Spells.cpp

```diff
@@ -445,6 +461,9 @@ void SpellStats::Initialize() {
     }
 }
 
+/**
+ * TODO: this function only called from EVENT_CastSpell processing, but such event is never used in game
```

Comment:
This is used for at least the firebolts in Temple of the Moon off the top of my head.

Reply:
Yeah let's fix the comment.

Reply:
Does that mean that events for EventProcessor functions are queued somewhere in game scripts?

Reply:
Okay I changed comment to something I think that this function do.
Correct me if I am mistaken in something.